jsapp/js/components/modalForms/submission.es6

```diff
@@ -500,18 +517,25 @@ class Submission extends React.Component {
           </div>
 
           <div className='submission-actions'>
-            {this.userCan('change_submissions', this.props.asset) && this.state.enketoEditLink &&
-              <a href={this.state.enketoEditLink}
-                onClick={this.promptRefresh}
-                target='_blank'
-                className='mdl-button mdl-button--raised mdl-button--colored'>
-                {t('Edit')}
+            {this.userCan('change_submissions', this.props.asset) &&
+              <a
+                onClick={this.launchEditSubmission.bind(this)}
+                className='mdl-button mdl-button--raised mdl-button--colored'
+                {...({
```

Comment:
:cool:, made me learn something :)
For tidiness, could this and the following two lines be replaced with `{...{disabled: this.isSubmissionEditable() ? null : true}}`?

Reply:
I don't remember why I did it this way honestly :-D Changed to even simpler solution.

Reply:
:smile: I think I know why you did it: because HTML wants either `<tag>` or `<tag disabled>`, not something like `<tag disabled="true">`. I think React [handles this well](https://github.com/facebook/react/issues/961), though.

I was still puzzled in a different way because `<a>` is not supposed to recognize `disabled`, but yet it works here (and not in any simple HTML file I created). I see the reason why: https://github.com/kobotoolbox/kpi/blob/9db69cb7bcae52d8cb20092707e0b67df3a8e9cb/jsapp/scss/libs/_mdl.buttons.scss#L153.